### ISSUE TITLE
Integrate extra information from the physics roomfinder

### ISSUE THREAD

**Issue by CommanderStorm** [TUM-Dev/navigatum#34]
as mentioned in https://github.com/TUM-Dev/navigatum/blob/main/deployment/README.md we still need to tap into the potential datasources of the physics roomfinder

**Comment by CommanderStorm** [comment]
The NAT wants to shut down their Roomfinder by the end of March and institute a redirect to nav.tum.de

Given that 
- I currently am in my exam phase for the next few weeks and
- will be working on map overlays afterwards (Bachlor Thesis 😉),  

I am unsure if I can get this done in this time frame.

Do you (@octycs) have the resources to take over the work on this issue?

**Comment by octycs** [comment]
Yes, since I have finished my exams recently, I can take it over
